style/header.css

```diff
@@ -228,10 +210,30 @@
 .menu-items li {
   padding: 10px 20px 10px;
 }
-#boilers:hover .sup-menu.menu-items-lvl2,
-#firetube:hover .sup-menu.menu-items-lvl3 {
-  visibility: visible;
-  opacity: 1;
+
```

Comment:
please split the menu CSS code into separate file "navigation.css"

Reply:
Done, but I have included the main menu in navigation.css, if I miss understand it I'm willing to change it.
![image](https://github.com/YAQ0T/XNGAGE-first-project-html-css/assets/63197328/16dd5bd0-a07a-4ce7-9e68-d64c3c7c5f96)